src/mmw/apps/modeling/mapshed/tasks.py

```diff
@@ -319,6 +318,43 @@ def avg_awc(result):
     }
 
 
+
+@shared_task(throws=Exception)
```

Comment:
This task has too many blank lines above it, causing the build to be marked as unstable.